### ISSUE TITLE
gowns--pajamas

### ISSUE THREAD

**Issue by fhxshrs** [rdrg109/npcr-subtitles#34]
https://github.com/rdrg109/npcr-subtitles/blob/6cb024dfc2e3453d1fb5674eae17cf5b187b8c89/sentences/44-1.yaml#L14C19-L14C24

**Comment by rdrg109** [comment]
@fhxshrs Estoy de acuerdo contigo, la traducción debería ser "pajamas", en vez de "gowns". Ya hice los cambios: 5c2d074

De acuerdo a [este diccionario en inglés](https://www.ldoceonline.com/dictionary/gown), "gown" significa "a long dress that a woman wears on formal occasions". Estas son las primeras imagenes que muestra Google Images cuando busco esa palabra.

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/531dbe65-413a-4a78-8c13-f8ef9d64f3e7)

Cuando busco "chinese gown", Google muestra estas imágenes.

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/f40d54cd-7a33-4726-a2ea-9b327c4f0ad9)

En el vídeo, el señor parece sostener un vestido que es muy similar a las imagenes que vemos cuando buscamos "chinese gown" en Google, entonces tengo esta pregunta: ¿Lo que sostiene el señor es una "pajama" o es un "chinese gown"?

![image](https://github.com/rdrg109/subtitles-npcr/assets/74389646/35998764-4251-4d39-a0c8-55df3d495098)

**Comment by fhxshrs** [comment]
首先，我认为还是应该是"pajama"。你提到的"Chinese gown"应该就是“旗袍”，而旗袍具有修身的特点，一般也只会在比较正式的场合穿（但其实现代中国并不是这样）。课文中这位老板在卖的是睡衣，睡衣的特点是宽松舒适。而视频里所展示的只是刚好看起来和你搜索到的旗袍颜色、材质相似，但细看并非同一种款式。

**Comment by rdrg109** [comment]
明白了！在我的眼里老板拿着的东西是旗袍，通过你的评论我了解到不是旗袍而是睡衣。
